db/migrate/20221111152908_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,20 @@
+class AddConfirmableToDevise < ActiveRecord::Migration[7.0]
+  def up
+    add_column :users, :confirmation_token, :string
+    add_column :users, :confirmed_at, :datetime
+    add_column :users, :confirmation_sent_at, :datetime
+    add_column :users, :unconfirmed_email, :string # Only if using reconfirmable
```

Comment:
- [ ] When creating migrations I got this error about `reconfirmable`, kindly fix this so the migrations are created correctly

![Screenshot_20221111_184612](https://user-images.githubusercontent.com/25227022/201398836-deaa56f8-afff-4b2b-88da-5bbe00d929fd.png)